src/gradletestrunnersmoketest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+gradletestrunnersmoketest () {
+
+  if [[ -n "${google_project}" ]] && [[ "${google_application_credentials}" != "" ]] && [ -f ${google_application_credentials} ]; then
+    export GOOGLE_APPLICATION_CREDENTIALS=${google_application_credentials}
+    export GOOGLE_CLOUD_PROJECT=${google_project}
+    export GOOGLE_CLOUD_DATA_PROJECT=${google_data_project}
```

Comment:
I think we are reexporting these variables multiple times. It may be better to simply define defaults for them in the action.

Reply:
This is following the convention set up in the int test step, and I think we can address this in a follow up ticket.